bioconda_utils/pkg_test.py

```diff
@@ -164,6 +164,10 @@ def test_package(
         raise RuntimeError('internal involucro wrapper missing')
     cmd += ['--involucro-path', involucro_path]
```

Comment:
Ah, I forgot that we already use that `involucro` wrapper to inject `POSTINSTALL` for the activation scripts...
Well, then we can also directly set `PREINSTALL` there, of course!